docs/apache-airflow/howto/operator/external_task_sensor.rst

```diff
@@ -53,6 +53,16 @@ via ``allowed_states`` and ``failed_states`` parameters.
     :start-after: [START howto_operator_external_task_sensor]
     :end-before: [END howto_operator_external_task_sensor]
 
+ExternalTaskSensor with task_group dependency
+---------------------------------------------
+In Addition, we can also use the :class:`~airflow.sensors.external_task.ExternalTaskSensor` to make tasks on a DAG
+wait for another ``task_group`` on a different DAG for a specific ``execution_date``.
```

Comment:
The sensor still names the argument `execution_date_fn` so I guess using `execution_date` here is OK.

Reply:
right!

Reply:
let me have something like **`execution_date/logical_date`** to avoid any confusions.